### PR TITLE
fix card issues on documentation platform

### LINKED ISSUE #941: Card issues
## Preflight Checklist

* [X] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/.github/CONTRIBUTING.md) for this project.
* [X] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/.github/CODE_OF_CONDUCT.md) that this project adheres to.
* [X] I have searched the issue tracker for an issue that matches the one I want to file, without success.

## Which package(s) are you using?

- [X] `lyne-components`
- [ ] `non package issue`

## Detailed description

> Documentation platform issues:
- Wrong positioning of card-badge
- camelCase issues with properties
- Styling issues / component background colors

### PR DESCRIPTION
**Pull request by 4aficiona2**
## Preflight Checklist

* [X] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/.github/CONTRIBUTING.md) for this project.
* [X] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/.github/CODE_OF_CONDUCT.md) that this project adheres to.
* [X] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #941 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [X] Build (`npm run build`) was run locally and any changes were pushed
- [X] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Changes

Changes in this pull request:

- Fixes documentation platform issues

## Browsers

I tested the build on the following browsers:
- [x] Firefox Desktop
- [x] Chrome Desktop
- [x] Safari Desktop
- [ ] Firefox Mobile
- [ ] Chrome Mobile
- [ ] Safari Mobile
- [x] Opera
- [x] Brave

## Pull request type

Please check the type of change your PR introduces:
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

Fixes #941 

@lyne-design-system/core

### CODE REVIEW

File: src/components/lyne-card-product/lyne-card-product.stories.js [hunk 1]
```diff
@@ -21,6 +21,24 @@ const wrapperStyle = (context) => {
   return `background-color: ${ColorMilkDefault};`;
 };
 
+/* ************************************************* */
+/* Documentation platform container                  */
```
Comment (feerglas) [inline src/components/lyne-card-product/lyne-card-product.stories.js:25]: I guess you can remove this comment block?

**Review by feerglas — APPROVED** [review]
Just general information: the option for container styles for the documentation platform is documented in `STORYBOOK.md` in the root of our project.

File: src/components/lyne-card-product/lyne-card-product.stories.js (hunk 1 above)
Comment (4aficiona2) [reply src/components/lyne-card-product/lyne-card-product.stories.js:25]: I would like to introduce this change in a subsequent commit/merge with master. For this reason it is commented out.

File: src/components/lyne-card-product/lyne-card-product.stories.js [hunk 2]
```diff
@@ -636,6 +656,20 @@ CardProductTheWholeShabang.args = {
 };
 
 CardProductTheWholeShabang.documentation = {
+
+  /*
```
[the remark this answers is not in the document]
Comment (4aficiona2) [reply src/components/lyne-card-product/lyne-card-product.stories.js:660]: Corresponds to the comment above and will be introduced in a future commit/merge.

**Comment by 4aficiona2** [comment]
@feerglas Thanks for the review and feedback. I missed the storybook documentation markdown file but this one perfectly describes how to introduce additional container styles. 👍

**Pull request merged**